src/m365/aad/commands/group/group-list.ts

```diff
@@ -0,0 +1,60 @@
+import * as chalk from 'chalk';
+import { Logger } from '../../../../cli';
+import {
+  CommandOption
+} from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import { GraphItemsListCommand } from '../../../base/GraphItemsListCommand';
+import commands from '../../commands';
+import { Group } from './Group';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  output?: string;
+}
+
+class AadGroupListCommand extends GraphItemsListCommand<Group>   {
+  public get name(): string {
+    return commands.GROUP_LIST;
+  }
+
+  public get description(): string {
+    return 'Lists all Azure AD groups in the tenant.';
+  }
+
+  public getTelemetryProperties(args: CommandArgs): any {
+    const telemetryProps: any = super.getTelemetryProperties(args);
```

Comment:
Since we're not tracking any additional telemetry options on this command, we can remove `getTelemetryProperties` from this class.